docs/index.md

```diff
@@ -35,11 +35,11 @@ In this section we cover installing Ginkgo, Gomega, and the `ginkgo` CLI.  We bo
 
 ### Installing Ginkgo
 
-Ginkgo uses [go modules](https://go.dev/blog/using-go-modules).  To add Ginkgo to your project, assuming you have a `go.mod` file setup, just `go get` it:
+Ginkgo uses [go modules](https://go.dev/blog/using-go-modules).  To add Ginkgo to your project, assuming you have a `go.mod` file setup, just `go install` it:
 
 ```bash
-go get github.com/onsi/ginkgo/v2/ginkgo
-go get github.com/onsi/gomega/...
+go install -mod=mod github.com/onsi/ginkgo/v2/ginkgo
+go install -mod=mod github.com/onsi/gomega/...
```

Comment:
do we need to `go install` gomega - since it's a library should we not be `go get`ting it?

Reply:
Oops--you're right. If Gomega doesn't need anything compiled, then `go get` is more appropriate and installs the most up-to-date version (which does not appear to be true for `go install`). I'll update the PR.